ch.yaml

```diff
@@ -158,6 +161,19 @@ methods:
         date += 7
       end
       date
+  ch_be_zibelemaerit:
+    # fourth Monday in november
+    arguments: year
+    ruby: |
+      date = Date.civil(year,11,1)
+      # Find the first Monday of September
```

Comment:
@hrigu `# Find the first Monday of `**November**?

Reply:
@hrigu I think so, but how is it?

Reply:
yes, this is correct. Thank you